packages/gatsby-plugin-feed/README.md

```diff
@@ -22,6 +22,25 @@ plugins: [
 ]
 ```
 
+To complete the feed setup, you need to expose a GraphQL entry for our content called `fields.slug` by modifying `gatsby-node.js`. Start with the following code, noting the reference to `MarkdownRemark` content. For content sources other than Markdown, you will want to modify it:
```

Comment:
What do you think about linking to the newly established doc you're creating here?

Generally - this makes it seem like you _have_ to create a slug, when rather you could just customize the query to match whatever data structure your posts require? (e.g. path from wordpress, something else from drupal, etc.)

Reply:
I can link to the new doc, for sure. The examples given in the plugin README don't work on their own without more details, though. Perhaps I can reword it to be more open-ended?

Reply:
Yeah, really good point! Our README's should always show a working case, even if it's slightly trivial. This one is kind of hard because the developer will _have_ to implement this section depending on how they're sourcing the data.

But yeah - I like your idea of re-wording maybe. As in, "Here's an example of how you could implement this plugin with Markdown, but for other sources, you will need a way to uniquely identify content--typically a URL or slug."

or something!